### PR TITLE
Actualize

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | 

Related: https://github.com/yiisoft/demo/pull/471

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +32,18 @@ Configure HTTP client (usually that is `config/common/sentry.php`):
...
 return [
-    \Http\Client\HttpClient::class => \GuzzleHttp\Client::class,
-    \Http\Client\HttpAsyncClient::class => [
-        'class' => \Http\Adapter\Guzzle7\Client::class,
+    HttpClient::class => GuzzleClient::class,
+    HttpAsyncClient::class => [
+        'class' => GuzzleClientAdapter::class,
         '__construct()' => [
-            \Yiisoft\Factory\Definition\Reference::to(\GuzzleHttp\Client::class),
+            Reference::to(GuzzleClient::class),
```
Comment (samdark) [inline README.md]: Can't it be `HttpClient::class`? And if so, this will be assumed without explicit configuration.

**Review by samdark — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xepozz) [reply README.md]: The client accepts an object, string class name is not possible here afaik

Comment (samdark) [reply README.md]: OK but why `Reference::to(GuzzleClient::class)` instead of `Reference::to(HttpClient::class)`?

Comment (arogachev) [reply README.md]: Do you mean it should be like that even before these changes?

Comment (arogachev) [reply README.md]: I tested and it both variants work fine. Which one should be kept?

Comment (samdark) [reply README.md]: HttpClient::class

Comment (xepozz) [reply README.md]: `Http\Client\HttpClient` didn't work last time because of incompatibility. I hope it has changed now.

Comment (arogachev) [reply README.md]: OK. Fixed here and demo too.

**Pull request merged**